poll/views.py

```diff
@@ -23,11 +23,25 @@ def home(request):
     up_movers = ranks.order_by('-ppv_diff')[0:5]
     down_movers = ranks.order_by('ppv_diff')[0:5]
 
+    dropped = []
```

Comment:
This code is basically duplicated in poll_viewer() and home(). Would you rather me pull it out into a helper function?

Reply:
I think I like the idea of having them be different views. The version in poll_viewer has slightly different information than the one on home by design. There's enough overlap that I could see some of it being brought into a helper function, though.